script/generatecert

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+set -x
+
+mkdir cert
```

Comment:
I actually think it's useful to keep all of the sensitive information in a single directory, especially when running locally. Just added the directory to the `.gitignore`, too. Let me know if you still want me to change it.